.github/workflows/regressions.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 MosaicML LLM Foundry authors
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import datetime
+import os
+import subprocess
+
+dir_path = os.path.dirname(os.path.abspath(__file__))
+regressions_dir = os.path.join(dir_path, 'regression_yamls')
```

Comment:
Is there a style rule that global variables should be capital case?

Reply:
not sure. This file might not get linted, not sure. can make caps tho